scipy/stats/utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import (
```

Comment:
What I meant was to put these in `scipy._util` so that these can be used in all modules.